_includes/pbc/all/install-features/202304.0/install-the-shipment-feature.md

```diff
@@ -233,6 +371,43 @@ class ShipmentDataImportConfig extends SprykerShipmentDataImportConfig
 }
 ```
 
+**src/Pyz/Zed/ShipmentTypeDataImport/ShipmentTypeDataImportConfig**
+```php
+<?php
+
+namespace Pyz\Zed\ShipmentTypeDataImport;
+
+use Generated\Shared\Transfer\DataImporterConfigurationTransfer;
+use Spryker\Zed\ShipmentTypeDataImport\ShipmentTypeDataImportConfig as SprykerShipmentTypeDataImportConfig;
+
+class ShipmentTypeDataImportConfig extends SprykerShipmentTypeDataImportConfig
+{
+    /**
+     * @return \Generated\Shared\Transfer\DataImporterConfigurationTransfer
+     */
+    public function getShipmentTypeDataImporterConfiguration(): DataImporterConfigurationTransfer
+    {
+        return $this->buildImporterConfiguration('shipment_type.csv', static::IMPORT_TYPE_SHIPMENT_TYPE);
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\DataImporterConfigurationTransfer
+     */
+    public function getShipmentTypeStoreDataImporterConfiguration(): DataImporterConfigurationTransfer
+    {
+        return $this->buildImporterConfiguration('shipment_type_store.csv', static::IMPORT_TYPE_SHIPMENT_TYPE_STORE);
+    }
+    
+    /**
+     * @return \Generated\Shared\Transfer\DataImporterConfigurationTransfer
+     */
+    public function getShipmentMethodShipmentTypeDataImporterConfiguration(): DataImporterConfigurationTransfer
+    {
+        return $this->buildImporterConfiguration('shipment_method_shipment_type.csv', static::IMPORT_TYPE_SHIPMENT_METHOD_SHIPMENT_TYPE);
+    }
+}
+```
+
```

Comment:
The following plugin is outdated in the IG:
* MoneyCollectionFormTypePlugin => \Spryker\Zed\MoneyGui\Communication\Plugin\Form\MoneyCollectionFormTypePlugin.
*